pkg/proxy/proxy.go

```diff
@@ -158,14 +158,17 @@ func isJavaInstalled() bool {
 
 // to extract ca certificate to temp
 func ExtractCertToTemp() (string, error) {
+	// Set the umask to allow access to the file for all users
+	err := utils.SetUmask(0)
```

Comment:
why are you changing the permission of the cert file, nobody tries to access that. It is not needed

Reply:
Previously there was an implementation of chmod command here,which is why i had changed the permissions here. Reverting as you deem it unnecessary.